src/main/resources/application.yml

```diff
@@ -5,11 +5,32 @@ integration:
   byggr:
     connect-timeout-in-seconds: 10
     read-timeout-in-seconds: 60
+
 service:
   byggr:
     filter-util:
       applicant:
         roles: SOK
+
+neighborhood-notification-template:
+  byggr-files:
+    sub-directory: /files/
+
+cache:
+  enabled: true
+  caches:
```

Comment:
Good point, but I think I'll go for the comment option.

(Note: I don't know how Eclipse works, but IntelliJ introspects the corresponding configuration property, and if a default value exists it is shown as an auto-complete option, leaving out the need for specifying repeated values instead of using configured defaults.)